### PR TITLE
Overall page structure

### PR DESCRIPTION
**Pull request by starshaped**
* Cleaned up HTML
* Added gulp to compile Sass
* Added instructions in README to set up gulp compilation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,14 @@ This is the codebase for the NERD Summit website: https://nerdsummit.org/
 
 The site is essentially a one-page website with most of the content in `index.html` and with `code.html` being the Code of Conduct page. 
 
-No SASS is used, just CSS.
+In the old version of the site, no SASS is used, just CSS.
```
Comment (claytron5000) [inline README.md]: delete

File: sass/base/_reset.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+/* Make images easier to work with */
+img {
+  max-width: 100%;
+  display: block;
+}
+
+/* Natural flow and rhythm in articles by default */
+article > * + * {
```
Comment (claytron5000) [inline sass/base/_reset.scss]: lobotomized owl! https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/

File: README.md (hunk 1 above)
Comment (starshaped) [reply README.md]: Done and done! Deleted that line.

File: README.md [hunk 3]
```diff
@@ -5,10 +5,14 @@ This is the codebase for the NERD Summit website: https://nerdsummit.org/
...
-No SASS is used, just CSS.
-
 The sessions section is built with Javascript via the `/js/sessions.js` file which gets the sessions data from `data/sessions.json`. This is currently in the process of being changed to get data from a Google Sheet via a JSON feed.  See BACKEND section.
 
+# FRONTEND
+
+Starting with the 2020 version of the site, Gulp has been configured to use Sass and PostCSS. In order to use this, install everything by opening a terminal window, navigating to the `nerdsummit2020` folder on your machine, and running `npm install`. To run the Gulp command, run `gulp watch`.
+
+CSS classes should be named by using the BEM (Block, Element, Modifier) methodology. To learn more, [Get BEM](http://getbem.com/) give an overview of the methodology, ways to implement it, and answers frequently asked questions.
+
```
Comment (starshaped) [inline README.md]: Added this paragraph about using BEM. Let me know if I need to add any more information about it!

**Pull request merged**